### PR TITLE
Password response

### PR DESCRIPTION
**Pull request by ivan-unfolds**
@rjmk the app is showing a successful response when the password is changed. With this implementation you can only do it once (you have to sign out if you want to do it again). Mostly because I can't think where to reset this bit of state. What do you think? Thinking this might not be such a bad idea.

### CODE REVIEW

File: src/user/redux/modules/change_password.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
-import { FETCHED_MEMBER } from '../../../shared/redux/modules/member.js'
+import { createAction } from 'redux-actions'
```
Comment (rjmk) [inline src/user/redux/modules/change_password.js:3]: Let's keep flow on every module. Add a `/* @flow */` annotation to the top and see if you can resolve the errors (you'll need to add a couple of type signatures)

Comment (ivan-unfolds) [inline src/user/redux/modules/change_password.js:3]: Will get on with this now. I was wondering though, what benefits does flow provide? I don't think I have experienced them...

Comment (rjmk) [inline src/user/redux/modules/change_password.js:3]: It's like tests. It makes sure everything glues together correctly. If you use it a bit more widely than we do, you can get a fair bit of confidence that when you make a change to a function signature in one place, you'll have correctly changed all the places that consume it

Comment (ivan-unfolds) [inline src/user/redux/modules/change_password.js:3]: That sounds good, not sure I grasp it fully though. Will have to do some background reading.

**Pull request merged**